CADETProcess/optimization/__init__.py

```diff
@@ -83,3 +90,33 @@
 from .optimizer import *
 from .scipyAdapter import COBYLA, TrustConstr, NelderMead, SLSQP
 from .pymooAdapter import NSGA2, U_NSGA3
+
+import importlib
+
+try:
+   from .axAdapater import GPEI, BotorchModular, NEHVI
+   from botorch.models.gp_regression import FixedNoiseGP
+   from botorch.acquisition import (
+      qExpectedImprovement,
```

Comment:
Why are these modules imported here? Should they be exposed to CADET-Process users?